### ISSUE TITLE
JSON encoding error: PhaseEncodingAxis "i" -> 0+1i

### ISSUE THREAD

**Issue by HenkMutsaerts** [ExploreASL/ExploreASL#306]
The PhaseEncodingAxis "i" is recognized incorrectly as numerical (imaginary number, 0+1i) instead of a char array "i", either by `dcm2niiX` or by our own `ExploreASL_Import: parms2json` functionality? This char is meant to denote that axis "i" is used for phase encoding (rather than "j" or "k").

* I will fix this in the ASL and M0 sidecars in `/External/TestDataSet/`
* @MDijsselhof you observed this as well right?
* @jan-petr can you fix this?

**Comment by MichaelStritt** [comment]
Hey, I just checked it out.

I ran a test on Windows 10. We read in the **PhaseEncodingAxis** in `xASL_adm_LoadParms` (line 135) using `spm_jsonread`:

```
JSONParms = spm_jsonread(JSONPath);
```

I tested the datatype and got:

```
isnumeric(JSONParms.PhaseEncodingAxis)
ans = 0
JSONParms.PhaseEncodingAxis=='i'
ans = 1
```

So in my case the variable is definitely a **char** already. Is it possible that the spm_jsonread import works differently on different operating systems? I added a minor fix (dbf82b8 ) which should make it work for you, but I couldn't reproduce the error on windows.
